.github/workflows/sca.yml

```diff
@@ -63,7 +63,7 @@ jobs:
 
       - name: Dependencies
         working-directory: ./
-        run: npm ci --legacy-peer-deps
+        run: npm ci
```

Comment:
I wasn't as sure about removing the flag here as I'm not entirely sure what this will run on.